### ISSUE TITLE
poetry doesn't work with pip==9.0.2

### ISSUE THREAD

**Issue by playpauseandstop** [sdispater/poetry#17]
Hi,

After upgrading `pip` to 9.0.2 version on MacOS 10.11.6 using Python 3.6.4 installed via `pyenv` `poetry` stops work with,

```
$ poetry --version
Traceback (most recent call last):
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/bin/poetry", line 7, in <module>
    from poetry import console
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/poetry/__init__.py", line 1, in <module>
    from poetry.console import Application
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/poetry/console/__init__.py", line 1, in <module>
    from .application import Application
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/poetry/console/application.py", line 8, in <module>
    from poetry.poetry import Poetry
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/poetry/poetry.py", line 13, in <module>
    from .repositories.pypi_repository import PyPiRepository
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/poetry/repositories/pypi_repository.py", line 2, in <module>
    from pip.req import InstallRequirement
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pip/__init__.py", line 45, in <module>
    from pip.vcs import git, mercurial, subversion, bazaar  # noqa
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pip/vcs/mercurial.py", line 9, in <module>
    from pip.download import path_to_url
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pip/download.py", line 40, in <module>
    from pip._vendor import requests, six
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pip/_vendor/requests/__init__.py", line 98, in <module>
    from . import packages
  File "/Users/playpauseandstop/.pyenv/versions/3.6.4/lib/python3.6/site-packages/pip/_vendor/requests/packages.py", line 12, in <module>
    sys.modules['pip._vendor.requests.packages.' + mod] = sys.modules["pip._vendor." + mod]
KeyError: 'pip._vendor.urllib3.contrib'
```

In same time, pip seems to work as usual, it installed packages and able to execute common commands (`freeze`, `list`, etc),

```
$ pip freeze | grep poetry
poetry==0.6.1
$ pip list -lo
Package                Version Latest Type 
---------------------- ------- ------ -----
decorator              4.1.2   4.2.1  wheel
http-prompt            0.10.2  0.11.2 wheel
MacFSEvents            0.7     0.8.1  sdist
pexpect                4.3.1   4.4.0  wheel
pipenv                 9.0.1   11.8.0 sdist
pipenv-to-requirements 0.1.9   0.3.1  wheel
python-dateutil        2.6.1   2.7.0  wheel
virtualenv-clone       0.2.6   0.3.0  wheel
```

----

Related issues: https://github.com/pypa/pip/issues/5079 + https://github.com/pypa/pip/issues/5081

**Comment by sdispater** [comment]
This should be fixed in the latest release (`0.6.2`)

**Comment by playpauseandstop** [comment]
Can confirm.

Thanks
